src/main/java/io/kestra/plugin/gcp/bigquery/Query.java

```diff
@@ -629,7 +629,8 @@ private Object convertCell(Field field, FieldValue value, boolean isRepeated) {
         }
 
         if (LegacySQLTypeName.DATETIME.equals(field.getType())) {
-            return Instant.parse(value.getStringValue() + "Z");
+            Instant.parse(value.getStringValue() + "Z");
```

Comment:
You can remove this line since you just return a String just behind, else you want to try catch it to do something in case an exception is caught?